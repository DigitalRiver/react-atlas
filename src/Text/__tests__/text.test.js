import React from "react";
import { mount } from "enzyme";
import { Text } from "../index";
import renderer from "react-test-renderer";

describe("Test Text component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Text>Hello World!</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Text component - Body Text", function() {
    const comp = mount(<Text body>Body Text</Text>);
    expect(comp.props().body).toBe(true);
    expect(comp.props().as).toEqual("span");
  });
});
