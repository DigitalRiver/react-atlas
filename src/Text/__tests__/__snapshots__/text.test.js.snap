// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Text component renders correctly 1`] = `
<span
  className="defaultFont"
  htmlFor={undefined}
  id={undefined}
  style={undefined}
>
  Hello World!
</span>
`;
